frontend/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -6,13 +6,22 @@ import Riders from './Riders';
 import Home from './Home';
 import Locations from './Locations';
 import Settings from './Settings';
+import DriverDetail from '../../components/DriverDetail/DriverDetail';
 
 const Dashboard = () => (
   <Router basename="/dashboard">
     <Sidebar>
       <Switch>
         <Route path="/home" component={Home} />
-        <Route path="/drivers" component={Drivers} />
+        <Route
+          path="/drivers"
+          render={({ match: { url } }) => (
+            <>
+              <Route path={`${url}/`} component={Drivers} exact />
+              <Route path={`${url}/driver`} component={DriverDetail} />
+            </>
+          )}
```

Comment:
Instead of using `render`, is it possible to make a separate component that has a router in it with the basename = "/drivers"? Then these routes can be put within that component's router.

Reply:
I googled about nested router, but it seems that there can't be another router within a router. I put the component in `Drivers` and then the `DriverDetail` page somehow wouldn't load.

Reply:
Interesting. I thought I'd done something with a nested router in the summer. I'll approve this PR as is then.